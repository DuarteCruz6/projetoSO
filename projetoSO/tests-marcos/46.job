WAIT 3000
WAIT 1000
WRITE [(k3,17)(y8,50)]
WRITE [(d2,18)(v9,30)(y9,93)(n2,15)]
WAIT 2000
WRITE [(i4,43)]
READ [y8]
WRITE [(c6,36)(r5,20)]
WRITE [(t6,79)(f7,30)(t5,27)(z3,76)]
WAIT 1000
WRITE [(u8,76)(v0,22)(b6,30)(g5,27)(e7,19)]
WRITE [(e1,97)(r9,62)(g3,61)(m2,59)]
SHOW
BACKUP
WRITE [(r6,47)(h2,64)(i1,14)(q9,67)]
READ [b6]
SHOW
WAIT 2000
READ [q9]
WRITE [(t2,58)(e6,51)]
READ [n2]
WRITE [(e8,7)(a0,77)(z0,7)(t0,77)(s8,35)]
READ [v9]
SHOW
READ [r6]
DELETE [r6,k3,g3]
BACKUP
WRITE [(x3,6)]
SHOW
SHOW
READ [y8]
WRITE [(l3,34)(s3,13)(b8,96)(j8,28)]
DELETE [s8,r5,s3,t5]
WRITE [(p8,63)(b2,71)(u2,24)(f8,89)(y1,6)]
WRITE [(m6,73)(i7,0)(a3,47)(z9,90)(p1,13)]
WRITE [(o3,2)(r4,8)(m4,3)(j0,29)]
READ [p1]
WRITE [(j9,99)]
DELETE [t2,b2,q9]
BACKUP
WRITE [(j4,31)]
SHOW
WRITE [(f4,15)(b4,76)(w6,37)(e2,70)(a2,39)]
DELETE [e8,j4,z3,f8,y9]
READ [o3]
DELETE [d2,f7]
WRITE [(q2,3)(u0,28)(d4,12)]
SHOW
SHOW
WRITE [(i0,5)(b9,1)(n8,5)(v6,9)(n9,72)]
WRITE [(o9,12)(l4,81)]
SHOW
WRITE [(h8,19)(v1,73)]
WRITE [(j3,96)(t3,49)]
WRITE [(d8,9)]
WRITE [(x8,52)(y0,26)]
BACKUP
DELETE [i0,w6]
WRITE [(p6,98)(a7,24)(w3,52)(q3,2)(q3,69)]
WRITE [(r0,69)(h0,9)(w8,23)(n0,49)]
DELETE [n8,e6,i4]
WRITE [(j5,89)(v2,38)(d9,41)]
WRITE [(z1,94)(o1,33)(f1,87)(c2,21)]
WAIT 1000
WRITE [(y4,94)]
READ [j5]
READ [a3]
