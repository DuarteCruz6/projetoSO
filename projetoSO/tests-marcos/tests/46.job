WRITE [(c0,52)]
READ [c0]
WRITE [(m0,46)(x6,28)]
WRITE [(k7,93)(j0,61)]
WRITE [(v5,57)(k6,90)(k1,100)(x4,28)]
WRITE [(a1,70)(d5,5)(y6,81)(s2,100)(o7,92)]
WRITE [(v0,90)(b2,0)(t2,73)(t3,81)]
WRITE [(f1,0)(h0,1)(h0,77)(x0,68)]
WRITE [(p9,76)(w9,41)]
WRITE [(w8,65)(l1,36)(u1,9)(p1,64)]
READ [p9]
READ [k1]
READ [y6]
WRITE [(p6,12)(j3,24)(r9,8)]
WRITE [(r7,47)(e1,42)(e4,59)]
DELETE [c0,l1,t3,y6]
WRITE [(n2,14)(e0,62)]
WRITE [(u4,64)(d4,52)(n6,3)(h9,39)]
READ [w8]
BACKUP
DELETE [u1,d4,e0,k1,h9]
WRITE [(s4,64)]
WRITE [(y9,9)(l4,12)]
WRITE [(a8,13)(u0,58)(w5,72)(x2,79)(w5,58)]
WRITE [(w4,8)]
DELETE [w5]
READ [j0]
WRITE [(v7,24)(g0,84)(d2,95)(f9,1)]
READ [u0]
WRITE [(y4,24)(j7,75)(i5,79)]
WRITE [(f0,28)(z0,18)(f3,28)(v3,73)(w6,4)]
WRITE [(m4,27)(t8,39)(n4,93)]
DELETE [x0,f1]
WRITE [(b0,41)]
DELETE [m4,k6,x4,j0,n6]
WRITE [(s0,26)(i7,43)(i2,15)]
WRITE [(s6,59)(o3,98)(c9,5)(t4,44)]
WRITE [(a0,66)(b4,99)(v9,97)]
READ [s2]
WRITE [(z1,14)]
DELETE [g0,p1,b0,f0,a1]
WRITE [(f0,28)(l1,13)(l3,39)(l7,36)(h5,93)]
DELETE [i5,x6,u4,a0,v9]
BACKUP
READ [h0]
WRITE [(b0,54)(t5,37)]
WRITE [(r2,32)(k0,58)(s3,72)]
DELETE [r9,v7,s4,h0]
WRITE [(r6,83)(j6,31)(b9,81)(o0,10)]
WRITE [(o6,78)(u9,70)(k6,16)]
WRITE [(a4,61)(i6,67)]
WRITE [(a7,99)(u4,72)(u8,16)(z3,13)]
READ [x2]
READ [f9]
DELETE [a4,m0,s3]
WRITE [(j1,49)(p0,65)(z8,27)(r8,65)(y1,68)]
DELETE [u8,t8,z1,w6]
WRITE [(e6,42)]
WRITE [(i1,53)(f6,83)(c8,31)(d1,1)]
READ [w4]
WRITE [(w2,22)(h4,38)]
READ [r8]
DELETE [i7,p9,l4,d5]
READ [o6]
BACKUP
