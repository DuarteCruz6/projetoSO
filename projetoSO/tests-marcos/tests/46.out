[(c0,52)]
[(p9,76)]
[(k1,100)]
[(y6,81)]
[(w8,65)]
[(j0,61)]
[(u0,58)]
[(s2,100)]
[(h0,77)]
[(x2,79)]
[(f9,1)]
[(w4,8)]
[(r8,65)]
[(o6,78)]
